Patch generated 20080922 by Phillip Warner <pc_warner@yahoo.com>
Permission granted to use this patch as you see fit.

--- Makefile.orig	2008-09-22 11:57:24.000000000 -0500
+++ Makefile	2008-09-22 11:57:51.000000000 -0500
@@ -46,7 +46,6 @@
 	make -C sesman install
 	make -C xup install
 	make -C docs install
-	if [ -d /etc/pam.d ]; then install instfiles/pam.d/sesman /etc/pam.d/sesman; fi
 	install instfiles/xrdp_control.sh $(DESTDIR)/xrdp_control.sh
 
 installdeb:
